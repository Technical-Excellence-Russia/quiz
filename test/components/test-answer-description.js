import React from "react"
import renderer from "react-test-renderer"

import AnswerDescription from "../../src/components/answer-description"

describe("Answer should", () => {

	let component = null

	beforeAll(() => component = renderer.create(
		<AnswerDescription>Test</AnswerDescription>
	))

	test("equal to snapshot", () => {
		expect(component.toJSON()).toMatchSnapshot()
	})

	afterAll(() => component.unmount())
})