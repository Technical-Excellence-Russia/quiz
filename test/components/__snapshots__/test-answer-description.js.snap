// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Answer should equal to snapshot 1`] = `
<div
  className="row"
>
  <div
    className="col m-2 p-2 text-left border"
    dangerouslySetInnerHTML={
      Object {
        "__html": "Test",
      }
    }
  />
</div>
`;
